src/routes/Root/Root.tsx

```diff
@@ -68,6 +78,44 @@ const Root = () => {
       <header role="banner" className="sticky top-0 z-10 print:hidden">
         <SkipLink hash="#jod-main" label={t('skiplinks.main')} />
         <NavigationBar
+          items={[
+            {
+              key: 'langSelector-key',
+              text: 'langSelector',
+              active: true,
+              component: ({ className }) => {
+                return (
+                  <div className="relative scale-90 mt-3">
+                    <RoundButton
```

Comment:
There is now an PR opened for the language button, you should be able to use the `onLanguageClick` prop for the language selection.
Opetushallitus/jod-design-system/pull/104